recipe/meta.yaml

```diff
@@ -52,28 +55,38 @@ outputs:
   - name: tbb-devel
     build:
       script: {{ build }} --no-rebuild --install-devel --install-docs
+      run_exports:
+        # excellent compatibility: https://abi-laboratory.pro/index.php?view=timeline&l=tbb
+        - {{ pin_subpackage('tbb', max_pin=None) }}
     requirements:
       build:
         - python *
         - {{ compiler('cxx') }}
         - m2w64-make                                     # [win]
+        - make                                           # [unix]
       run:
         - {{ pin_subpackage('tbb', exact=True) }}        # development package is for specific version of tbb
     test:
+      requires:
+        - {{ compiler('cxx') }}
+        - make                                           # [unix]
+      source_files:
+        - examples/test_all/fibonacci
       commands:
-        - if not exist %PREFIX%\Library\include\tbb\tbb.h exit 1  # [win]
-        - if not exist %PREFIX%\Library\lib\tbb.lib exit 1        # [win]
-        - test -f ${PREFIX}/include/tbb/tbb.h                     # [unix]
+        - cd examples\test_all\fibonacci                 # [win]
+        - set INCLUDE=%INCLUDE%;%LIBRARY_INC%            # [win]
+        - set LIB=%LIB%;%LIBRARY_LIB%                    # [win]
+        - nmake /f Makefile.windows                      # [win]
+        - make -C examples/test_all/fibonacci CXXFLAGS="-I$PREFIX/include -L$PREFIX/lib"  # [unix]
```

Comment:
The fact that conda-build requires include and library directories settings to be explicit was unexpected. Am I doing something wrong with the compiler in the test requirements?

Reply:
The Makefile doesn't make use of CPPFLAGS and uses only CXXFLAGS. Toolchain adds `-I$PREFIX/include` to CPPFLAGS and not CXXFLAGS.

Reply:
Thanks, I see. What's about the same question on Windows?

Reply:
INCLUDE and LIB are not set in https://github.com/AnacondaRecipes/aggregate/blob/master/vs2015/activate.bat . You'll have to open an issue in that repo asking them to set INCLUDE and LIB

Reply:
Thanks! Also, w.r.t. `CPPFLAGS` which is the pre-processor flags only.. what's supplying `-L` flag or sets `LIBRARY_PATH`?

Reply:
`LDFLAGS` should have `-L$PREFIX/lib`